### PR TITLE
[PR] Release preparation - 2.1.0

### PR DESCRIPTION
**Pull request by jeremyfelt**
Prepping for 2.1.0 release tomorrow. (It's been on my todo for a couple weeks now) 🙈 

Let me know if anything looks out of order. I'll be prepping a blog post as well.

### CODE REVIEW

**Comment by tomjn** [comment]
There's the new dashboard that got merged, and the VIP minimum coding standards PHPCS ruleset was added

**Comment by tomjn** [comment]
Also the .test TLD changes

**Comment by tomjn** [comment]
I think we can add a screenshot of the dashboard and new docs to the release post, and note the above things, then we'll be good to go. Don't forget to merge https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1300 and mention the search box on the dashboard and in the docs too

**Comment by jeremyfelt** [comment]
Sorry for the wait everyone, I'm going to pick this back up tonight.

**Comment by jeremyfelt** [comment]
* ✅  `vagrant provision` on existing VM works as expected. (Doesn't break) :)
* ✅  `vagrant destroy`, remove provisioned directories, `vagrant up` (so much fun)
* ✅  Release post written (local) for publish tomorrow. (still making some tweaks)
* ✅  This release branch prepped and ready for merge.

I'm going to plan on 17:00 UTC tomorrow for the 2.1.0 release. :shipit:

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,32 @@ title: Changelog
 permalink: /docs/en-US/changelog/
 ---
 
-## 2.x.x (TBD)
+## 2.1.0 (November 7, 2017)
+
+### Enhancements
+
+* Add cosmetic improvements to provisioning. This cleans up quite a bit of the junk that displayed on many lines when it should have displayed on one. See [#1247](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1247).
+* Update ack-grep to 2.16 See [#1148](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1148).
```
Comment (ocean90) [inline CHANGELOG.md]: The full stop is missing before "See".

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,7 +4,32 @@ title: Changelog
...
+* Dashboard (`http://vvv.test`) links now open in new tabs. See [#1168](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1168).
+* Speed up checking for `vvv-hosts` files in `Vagrantfile`. See [#1182](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1182).
+* Pass more configuration data to the VirtualBox VM on boot. See [#1197](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1197).
+* Link to `varyingvagrantvagrants.org`'s Add New Site in the `http://vvv.test` dashboard. See [#1220](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1220).
+* Build the VirtualBox VM name from the Vagrant directory and path hash. See [#1236](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1236).
+* PHPCS is now installed via Composer
+* VVV now uses .test instead of .dev for new installs and the dashboard
+* Added the VIP minimum coding standards to PHPCS. See [#1279](https://github.com/Varying-Vagrant-Vagrants/VVV/pull/1279)
```
Comment (ocean90) [inline CHANGELOG.md]: Lines 18-20 should all have a full stop.

**Comment by jeremyfelt** [comment]
Ok, let's get this started. Going to merge this into develop, push it to master, etc... and then get the release post up.

**Pull request merged**